lib/ansible/modules/cloud/cloudstack/cs_instance.py

```diff
@@ -20,128 +20,157 @@
 description:
     - Deploy, start, update, scale, restart, restore, stop and destroy instances.
 version_added: '2.0'
-author: "René Moser (@resmo)"
+author: René Moser (@resmo)
 options:
   name:
     description:
       - Host name of the instance. C(name) can only contain ASCII letters.
       - Name will be generated (UUID) by CloudStack if not specified and can not be changed afterwards.
       - Either C(name) or C(display_name) is required.
+    type: str
   display_name:
     description:
       - Custom display name of the instances.
-      - Display name will be set to C(name) if not specified.
-      - Either C(name) or C(display_name) is required.
+      - Display name will be set to I(name) if not specified.
+      - Either I(name) or I(display_name) is required.
+    type: str
   group:
     description:
       - Group in where the new instance should be in.
+    type: str
   state:
     description:
       - State of the instance.
+    type: str
     default: present
     choices: [ deployed, started, stopped, restarted, restored, destroyed, expunged, present, absent ]
   service_offering:
     description:
       - Name or id of the service offering of the new instance.
       - If not set, first found service offering is used.
+    type: str
   cpu:
     description:
       - The number of CPUs to allocate to the instance, used with custom service offerings
+    type: int
   cpu_speed:
     description:
       - The clock speed/shares allocated to the instance, used with custom service offerings
+    type: int
   memory:
     description:
       - The memory allocated to the instance, used with custom service offerings
+    type: int
   template:
     description:
       - Name, display text or id of the template to be used for creating the new instance.
       - Required when using I(state=present).
-      - Mutually exclusive with C(ISO) option.
+      - Mutually exclusive with I(iso) option.
+    type: str
   iso:
     description:
       - Name or id of the ISO to be used for creating the new instance.
       - Required when using I(state=present).
-      - Mutually exclusive with C(template) option.
+      - Mutually exclusive with I(template) option.
+    type: str
   template_filter:
     description:
       - Name of the filter used to search for the template or iso.
-      - Used for params C(iso) or C(template) on I(state=present).
+      - Used for params I(iso) or I(template) on I(state=present).
       - The filter C(all) was added in 2.6.
+    type: str
     default: executable
     choices: [ all, featured, self, selfexecutable, sharedexecutable, executable, community ]
     aliases: [ iso_filter ]
-    version_added: '2.1'
```

Comment:
keep quotes

Reply:
Quotes should be consistent, though. Single or double, whatever the Ansible standard is.